ProjectCoimbra.UWP/Project.Coimbra/Helpers/SongPagesHelper.cs

```diff
@@ -185,5 +184,31 @@ public static async Task FillListBoxAsync(ListBox listBox, string selectedValue,
                 nextButton.IsEnabled = true;
             }
         }
+
+        /// <summary>
+        /// Removes Item from list
+        /// </summary>
+        /// <param name="key">item key</param>
+        public static async Task RemoveFile(string key)
+        {
+            if (string.IsNullOrWhiteSpace(key))
+            {
+                throw new ArgumentNullException(nameof(key));
+            }
+
+            var midiFolder = await GetMidiFolderAsync(false).ConfigureAwait(true);
```

Comment:
Can you see if this would work with `.ConfigureAwait(false)`, which would be more efficient if it does?